test/proxy-upgrades/enable-2D-nonces.ts

```diff
@@ -193,7 +193,7 @@ describe("Upgradeability", function () {
 
     await expect(
       userSCW.connect(accounts[0]).updateImplementation(baseImpl5.address)
-    ).to.emit(userSCW, "ImplementationUpdated");
+    ).to.emit(userSCW, "ImplementationUpdated").withArgs(baseImpl.address, baseImpl5.address);
```

Comment:
ahaha nice thats why withArgs is important, won't ignore it next time when checking events